Scripts/update-translations.rb

```diff
@@ -124,27 +119,44 @@
   langs = ALL_LANGS
 end
 
+script_root = __dir__
+project_dir = File.dirname(script_root)
+
 langs.each do |code,local|
-  lang_dir = File.join('WordPress', 'Resources', "#{local}.lproj")
-  puts "Updating #{code}"
+  lang_dir = File.join(project_dir, 'WordPress', 'Resources', "#{local}.lproj")
+  puts "Updating #{code} in #{lang_dir}"
   system "mkdir -p #{lang_dir}"
 
-  # Download for production
-  system "if [ -e #{lang_dir}/Localizable#{strings_file_ext}.strings ]; then cp #{lang_dir}/Localizable#{strings_file_ext}.strings #{lang_dir}/Localizable#{strings_file_ext}.strings.bak; fi"
+  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
+  backup_destination = "#{destination}.bak"
+
+  # Step 2 – Download the new strings
+  if File.exist? destination
+    FileUtils.copy destination, backup_destination
+  end
 
   url = "#{download_url}/#{code}/default/export-translations?#{strings_filter}format=strings"
-  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
 
-  system "curl -fLso #{destination} \"#{url}\"" or begin
+  system "curl -fgLo #{destination} \"#{url}\"" or begin
```

Comment:
Ah good question:

- Removed `s` because it silences warnings, such as the one that was being emitted about glob syntax, which required
- Adding `g` to tell `curl` to ignore `[` and `]` in the URL rather than interpreting them as glob syntax

Once this moves into the toolkit, hopefully we can avoid using `curl` directly at all – that's been a source of _many_ issues in the past

Reply:
tbh at that point, why not use `net/http` and something like this (not tested, might contain typos):

```ruby
File.write(destination, Net::HTTP.get(URI.parse(url)))
```

_Note: There might be a potential gotcha with this solution as it reads the whole downloaded content in memory before writing it to disk, as opposed to streaming the write as the response body comes; but those files are <300KB so should not be an issue here._

Reply:
>Hmm... removing s will add quite a lot of noise to the output (I'm thinking about all the progress bars).
If we are only interested in errors, can we use sS?

Yes, this could certainly be worth a try! I can test it against the old code and see if it fails nicely for us.

As for using `net/http`, that's possible but as you mentioned @AliSoftware, potentially problematic – one big issue is that we have to follow redirects, which IIRC `Net::HTTP` doesn't do all that well?

IMHO ideally this code would live in the release toolkit (or a dependency of it?), and from there we could use something like the [HTTP gem](https://github.com/httprb/http) to do this work at a higher level that makes it easier to handle gotchas. 

I think for the purposes of this PR though I'd prefer to change it as little as possible 😅

Reply:
Resolved the `s` issue in f090c57efcedd83ea6ede1205719e41410a44e0c – I've just added it back for now – I feel like in the near-ish future we'll be replacing this anyway, so no point over-stressing about it.